docs/core/project-sdk/msbuild-props.md

```diff
@@ -1582,6 +1568,35 @@ When you use the [MSTest project SDK](../testing/unit-testing-mstest-sdk.md), th
 
 For more information, see [Microsoft.Testing.Platform profile](../testing/unit-testing-mstest-sdk.md#microsofttestingplatform-profile).
 
+## VSTest&ndash;related properties
+
+The following MSBuild properties are documented in this section:
+
+- [IsTestProject](#istestproject)
+- [TestingPlatformDotnetTestSupport](#testingplatformdotnettestsupport)
+- [TestingPlatformShowTestsFailure](#testingplatformshowtestsfailure)
+- [UseVSTest](#usevstest)
+- [MSTestAnalysisMode](#mstestanalysismode)
+
+### IsTestProject
+
+The `IsTestProject` property signifies that a project is a test project. When this property is set to `true`, validation to check if the project references a self-contained executable is disabled. That's because test projects have an `OutputType` of `Exe` but usually call APIs in a referenced executable rather than trying to run. In addition, if a project references a project where `IsTestProject` is set to `true`, the test project isn't validated as an executable reference.
```

Comment:
Being Exe feels more like an implementation detail of VSTest that can be misleading for users. @nohwnd What do you think?

Reply:
reading the section 3 times, I still don't get what it is trying to say, and I should be one of the experts 😁 I can see how some of the stuff is plausible, but there is no action point for the user, or any information on the high level meaning or "historical significance" of IsTestProject for VSTest (if I am not mistaking the property for some other property).

Reply:
In that case, maybe rewrite the whole paragraph to something like:

```
The `IsTestProject` property is set to true by `Microsoft.NET.Test.Sdk` NuGet package. It's used to decide whether a project is a VSTest test project or not, so that it's recognized by `dotnet test`
```